src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -107,6 +107,9 @@ The .NET Foundation licenses this file to you under the MIT license.
     <!-- Exclude System.IO.Compression.Native.dll -->
     <FrameworkAssemblies Include="$(IlcPath)\framework\*.dll" Exclude="$(IlcPath)\framework\*.Native.dll" />
 
+    <!-- Exclude simplified WindowsBase.dll when building WPF applications -->
+    <FrameworkAssemblies Remove="$(IlcPath)\framework\WindowsBase.dll" Condition="'$(UseWPF)' == 'true'" />
```

Comment:
WinForms applications will get WindowsBase from `Microsoft.WindowsDesktop.App` too. This will work for simple cases but it won't work for the more complex cases where the WinForms apps depends on the full WindowsBase.dll. It should be excluded unconditionally.

Reply:
so I will re-open previous PR and address your last feedback.